Dockerfile

```
@@ -11,5 +10,5 @@ LABEL "com.github.actions.description"="Wraps the Serverless Framework to enable
 LABEL "com.github.actions.icon"="zap"
 LABEL "com.github.actions.color"="red"
 
-RUN npm i -g serverless@1.54.0
+RUN npm i -g serverless@1.57.0
```

```suggestion
RUN npm i -g serverless@1.68.0
```
https://www.npmjs.com/package/serverless